### PR TITLE
Add 'Scalable Capital' and 'Baader Bank'

### PR DESCRIPTION
**Pull request by moriarty-jim**
Added fintech/asset manager "Scalable Capital Vermögensverwaltung GmbH" and bank "Baader Bank AG".

### CODE REVIEW

File: companies/baaderbank.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "at"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Baader Bank AG",
+    "address": "Weihenstephaner Straße 4\n85716 Unterschleißheim\nDeutschland",
+    "phone": "+49 89 5150 0",
+    "fax": "+49 89 5150 1111",
+    "email": "datenschutz@baaderbank.de",
```
Comment (baltpeter) [inline companies/baaderbank.json:14]: As this email is specific for privacy-related requests, we can set the `suggested-transport-medium` to `email`.

File: companies/scalablecapital.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "slug": "scalablecapital",
+    "relevant-countries": [
+        "de",
+        "at"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Scalable Capital Vermögensverwaltung GmbH",
+    "address": "Seitzstr. 8e\n80538 München\nDeutschland",
```
Comment (baltpeter) [inline companies/scalablecapital.json:11]: Let's use the contact details of the external DPO.

```suggestion
    "address": "c/o KINAST Rechtsanwaltsgesellschaft mbH\nHohenzollernring 54\n50672 Köln\nDeutschland",
```

File: companies/scalablecapital.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "phone": "+49 89 380 380 67",
```
Comment (baltpeter) [inline companies/scalablecapital.json:12]:
```suggestion
    "phone": "+49 221 222 183 0",
```

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record! I have requested some changes below.

**Comment by moriarty-jim** [comment]
Thanks for your feedback! I've updated the files.

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**